api.py

```diff
@@ -106,18 +106,19 @@ def record_track(self, track_info=None, replace=False):
         uri = track_info['uri']
         duration_ms = track_info['duration_ms']
         duration_s = int(duration_ms / 1000) + 1
-        filename = f"{track_info['name']} - {','.join([artist['name'] for artist in track_info['artists']])}"
+        filename = f"{track_info['name']} - {', '.join([artist['name'] for artist in track_info['artists']])}"
 
-        filename = filename.replace(" ", "_")
         filepath = f"songs/{filename}.mp3"
-        if os.path.exists(filepath):
+        exists = os.path.exists(filepath)
+        if exists:
             DINFO(f"An existing recorded file was found at {filepath}")
             # return True
 
-        if not filepath or replace:
+        if not exists or replace:
             os.system(f"./spotdl.sh {uri} \"{filename}\" {duration_s}")
             if not os.path.exists(filepath):
                 DERROR(f"\"{filepath}\" : Filepath does not exist. Exiting")
+                exit(2)
```

Comment:
I would rather `return false`, as if we are looping over a playlist, this would not terminate the whole process because of one error, while it would be better to continue on the next musics. However, this could be an option to add in the future.